src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -629,8 +629,8 @@ private async Task ValidateViewModel(EditTenantViewModel model, bool newTenant)
                 ModelState.AddModelError(nameof(EditTenantViewModel.RequestUrlPrefix), S["Host and url prefix can not be empty at the same time."]);
             }
 
-            var allOtherShells = allSettings.Where(tenant => !string.Equals(tenant.Name, model.Name, StringComparison.OrdinalIgnoreCase));
-            if (allOtherShells.Any(tenant => String.Equals(tenant.RequestUrlPrefix, model.RequestUrlPrefix?.Trim(), StringComparison.OrdinalIgnoreCase) && String.Equals(tenant.RequestUrlHost, model.RequestUrlHost, StringComparison.OrdinalIgnoreCase)))
+            var allOtherShells = allSettings.Where(t => !string.Equals(t.Name, model.Name, StringComparison.OrdinalIgnoreCase));
+            if (allOtherShells.Any(t => String.Equals(t.RequestUrlPrefix, model.RequestUrlPrefix?.Trim(), StringComparison.OrdinalIgnoreCase) && t.RequestUrlHost.Contains(model.RequestUrlHost, StringComparison.OrdinalIgnoreCase)))
```

Comment:
Single-letter variable names are harder to understand.

Reply:
We almost use letter if the entity is single like tenant, if it's compound like tenantsSettings we could use **ts**

Reply:
I really don't see why it would make sense to come up with single-use abbreviations. It only makes things easier to understand if we have proper variable names (especially that we had them here before) except for very specific cases used in the whole industry, like `i` as a loop index.

Reply:
We could suggest something in Coding Style for the entire repo, @agriffard who started that document

Reply:
Please feel free to do that. Otherwise, if you argue that in these two lines changing `tenant` to `t` is better for some reason then please do.

Reply:
Not an issues but I remember this from working with Microsoft in ASP.NET Core repos, especially localization